next.config.js

```diff
@@ -1,6 +1,10 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  publicRuntimeConfig: {
+    showVersion: process.env.SHOW_VERSION === 'true' || false,
```

Comment:
simplify to `process.env.SHOW_VERSION === 'true'`

Reply:
ohhh ya, okay